source/05-pages/article.stories.jsx

```diff
@@ -49,20 +51,29 @@ const articleDemoContent = `
   </p>
 `;
 
-const mainContent = wysiwygTwigTemplate({
-  content: articleDemoContent,
-});
-
 // For an example of customizing the content block on a demo page,
 // see Page.
-const articleContent = twigTemplate({
+const articleContent = args => twigTemplate({
+  ...args,
   title: 'As You Wish',
-  show_admin_info: false,
   show_footer: true,
+  date_format: 'medium-date',
+  year: {
+    long: '1987',
+  },
+  month: {
+    long: 'October',
+  },
+  day: {
+    short: '9',
+  },
   author_name: 'William Goldman',
-  body: mainContent,
+  content: articleDemoContent,
 });
 
-const Article = () => <PageWrapper>{parse(articleContent)}</PageWrapper>;
+const Article = args => <PageWrapper>{parse(articleContent(args))}</PageWrapper>;
```

Comment:
Codacy has a fix for the issue: [Function component is not a function expression](https://app.codacy.com/gh/forumone/gesso/pullRequest?prid=8642047)
```suggestion
var Article = function(args) {
```